### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by ineiti**
Changes to the comments - Linus' new way of doing code-review ;)

### CODE REVIEW

File: network/local.go [hunk 1]
```diff
@@ -26,16 +29,18 @@ func NewLocalRouterWithManager(ctx *LocalManager, sid *ServerIdentity) (*Router,
...
 type LocalManager struct {
 	// queues maps a remote endpoint to its packet queue. It's the main
-	//stucture used to communicate.
+	// stucture used to communicate.
 	queues map[endpoint]*connQueue
 	sync.Mutex
+	// The listening-functions used when a new connection-request arrives.
 	listening map[Address]func(Conn)
 
+	// Unique ID of this Local Manager.
```
Comment (nikkolasg) [inline network/local.go]: No it's not. It's more like a counter that we increment for each connection it creates/handle. So maybe rename the field to `counter` or stg.

**Comment by nikkolasg** [comment]
I think some comments on `connQueue` have to be changed so it reflects now that we are marshalling messages and passing only slice of bytes (no `Packet`)

**Comment by ineiti** [comment]
I propose we keep the `packet`-references, as they still _represent_ packets.

File: network/local.go [hunk 2]
```diff
@@ -25,17 +28,19 @@ func NewLocalRouterWithManager(lm *LocalManager, sid *ServerIdentity) (*Router,
...
 	// queues maps a remote endpoint to its packet queue. It's the main
-	//stucture used to communicate.
+	// structure used to communicate.
 	queues map[endpoint]*connQueue
 	sync.Mutex
+	// The listening-functions used when a new connection-request arrives.
 	listening map[Address]func(Conn)
 
-	baseUID uint64
+	// connection-counter for giving unique IDs for each connection.
```
Comment (nikkolasg) [inline network/local.go]: `... giving unique IDs to each connection` ? for vs to ?

File: network/local.go [hunk 3]
```diff
@@ -251,9 +265,10 @@ func (lc *LocalConn) Type() ConnType {
 	return Local
 }
 
-// connQueue is a struct that managers the message queue of a LocalConn.
-// Messages are pushed and retrieved FIFO style.
+// connQueue manages the message queue of a LocalConn.
+// Messages are pushed and retrieved in a FIFO.
```
Comment (nikkolasg) [inline network/local.go]: FIFO is not an object, but a method of ordering elements. A FIFO queue is an object. `...in a FIFO queue`

**Comment by nikkolasg** [comment]
Cool. Merging it.

**Pull request merged**